src/schema/v2/artist/insights.ts

```diff
@@ -86,6 +91,36 @@ export const ArtistInsights: GraphQLFieldConfig<any, ResolverContext> = {
       artist.highAuctionRecord = highAuctionRecord
     }
 
+    if (kind.includes("SOLO_SHOW")) {
```

Comment:
I see some repetition of what we have in `enrichWithArtistCareerHighlights`, maybe we could reuse something?

Reply:
the initial idea was to re-use `enrichWithArtistCareerHighlights`. thanks for flagging!